tests/tests_pytorch/trainer/connectors/test_signal_connector.py

```diff
@@ -78,8 +78,9 @@ def training_step(self, batch, batch_idx):
 
 @RunIf(skip_windows=True)
 @pytest.mark.parametrize("auto_requeue", (True, False))
-def test_auto_requeue_flag(auto_requeue):
-    trainer = Trainer(plugins=[SLURMEnvironment(auto_requeue=auto_requeue)])
+@pytest.mark.parametrize("requeue_signal", [(signal.SIGUSR1, signal.SIGUSR2, signal.SIGHUP)])
```

Comment:
```suggestion
@pytest.mark.parametrize("requeue_signal", [signal.SIGUSR1, signal.SIGUSR2, signal.SIGHUP])
```

This will fix the "expected int, not tuple" test error

Reply:
Oops missed that